08 - FractorCalculator.java

```diff
@@ -0,0 +1,103 @@
+import java.io.*;
```

Comment:
Nice job here using methods with parameters. I noticed you've used some Java functionalities that we haven't learned yet - objects, Exceptions, the buffered reader - unfortunately we won't be able to accept this assignment unless we have explanations of what those are and why you're using them - just since we haven't learned about them yet.

Reply:
As I told you before, in my previous school, we had a free program for those who wanted to learn to encode, and although we were taught little there, I remembered a few things that I sometimes use when encoding. In order to find the correct fraction in Java, we first need to specify the denominator and numerator. Abbreviated nomo , and deno .  Which, later, can be used to make it easier to calculate fractions using them.